src/local/room_name.rs

```diff
@@ -84,7 +87,11 @@ impl LocalRoomName {
 impl ops::Add<(i32, i32)> for LocalRoomName {
     type Output = Self;
 
-    /// Adds an (x, y) coordinate pair to this room name.
+    /// Offsets this room name by a given horizontal and vertical (x, y) pair.
+    ///
+    /// The first number offsets to the west when negative and to the east when
+    /// positive. The first number offsets to the north when negative and to
```

Comment:
second

Reply:
Would it be more natural to inverse the y direction to follow the cartesian standard?

Reply:
#200 :p

I would argue the game set this standard when they made `north` negative in in-room coordinates?

Reply:
Yeah, that's from game design. Pixels are counted from the top left 😞